package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "generate-hook": "nx workspace-generator open-feature hook",
-    "generate-provider": "nx workspace-generator open-feature provider",
+    "generate-hook": "nx generate open-feature hook",
+    "generate-provider": "nx generate open-feature provider",
```

Comment:
This was a breaking change. I had to change the commands to make them work in 16.